main.c

```diff
@@ -323,6 +329,47 @@ static bool ParseCommandLine(u8* argv[])
 		fprintf(stderr, "  Output to Pipe (%s)\n", g_Output_PipeName);
 		cnt	+= 2;
 	}
+	// default tcp output to stdout
+	if (strcmp(argv[0], "--output-tcp-stdout") == 0)
+	{
+		g_Output_TCP_STDOUT = true;
+		fprintf(stderr, "  Output TCP events to STDOUT\n");
+		cnt	+= 1;
+	}
+	// write tcp events to a named pipe
+	if (strcmp(argv[0], "--output-tcp-pipe") == 0)
+	{
+		g_Output_TCP_STDOUT = false;
+		g_Output_TCP_PipeName	= strdup(argv[1]);
+		fprintf(stderr, "  Output TCP events to Pipe (%s)\n", g_Output_TCP_PipeName);
+		cnt	+= 2;
+	}
+	// filter which tcp events are output
+	if (strcmp(argv[0], "--tcp-events") == 0)
+	{
+		if (!strstr(argv[1], "all"))
+		{
+			// We're filtering events, so turn them all off first
+			memset(&g_TCPEventFilter, 0, sizeof(g_TCPEventFilter));
+		}
+
+		if (strstr(argv[1], "netRTT"))
```

Comment:
Is this good enough, or should we have proper strict matching on the args, including an error if an unknown filter is passed, for eg.?
`strstr` would enable `netRTT,appRTT` even for `--tcp-events netRTTWhatever,appRTTtypo` etc. Is that good or bad?